specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerEffectiveConfigurationsList.json

```diff
@@ -12,51 +12,93 @@
-                "connectivityTopology": "HubAndSpokeTopology",
-                "hubId": "subscriptions/subscriptionB/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myHubVnet",
-                "deleteExistingPeering": true,
-                "isGlobal": true,
-                "appliesToGroups": [
-                  {
-                    "networkGroupId": "/subscriptions/subscriptionA/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/groups/group1",
-                    "useHubGateway": true,
-                    "groupConnectivity": "Transitive",
-                    "isGlobal": false
+            "id": "subscriptions/subscriptionA/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/connectivityConfigurations/myTestConnectivityConfig",
+            "properties": {
+              "displayName": "myTestConnectivityConfig",
+              "description": "Sample Configuration",
+              "connectivityTopology": "HubAndSpokeTopology",
+              "hubId": "subscriptions/subscriptionB/resourceGroups/myResourceGroup/providers/Microsoft.Network/virtualNetworks/myHubVnet",
+              "deleteExistingPeering": true,
+              "isGlobal": true,
+              "appliesToGroups": [
+                {
+                  "networkGroupId": "subscriptions/subscriptionA/resourceGroups/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/groups/group1",
+                  "useHubGateway": true,
+                  "groupConnectivity": "Transitive",
+                  "isGlobal": false
+                }
+              ],
+              "provisioningState": "Succeeded",
+              "configurationGroups": [
+                {
+                  "name": "group1",
+                  "id": "/subscriptions/subscriptionA/resourceGroup/myResourceGroup/providers/Microsoft.Network/networkManagers/testNetworkManager/networkGroups/group1",
+                  "type": "Microsoft.Network/networkManagers/networkGroups",
+                  "properties": {
+                    "displayName": "My Network Group",
+                    "description": "A group for all test Virtual Networks",
+                    "memberType": "VirtualNetwork",
+                    "groupMembers": [
+                      {
+                        "resourceId": "/subscriptions/subscriptionC/resourceGroup/rg1/providers/Microsoft.Network/virtualnetwork/vnet1"
+                      }
+                    ],
+                    "conditionalMembership": "",
+                    "provisioningState": "Succeeded"
                   }
-                ],
-                "provisioningState": "Succeeded"
-              }
+                }
+              ],
+              "region": "westus",
+              "commitTime": "2020-07-10T18:03:22.2578238+05:30"
+            }
+          },
+          {
+            "configType": "UserPolicy",
```

Comment:
Add an example for AdminPolicy?

Reply:
We don't need to add example for all situation here.